packages/contractkit/src/identity/metadata.ts

```diff
@@ -105,7 +106,11 @@ export class IdentityMetadataWrapper {
           throw new Error("Can't claim self")
         }
         break
-
+      case ClaimTypes.DOMAIN:
+        const existingClaims = this.data.claims.filter((el: any) => el.domain === claim.domain)
```

Comment:
Consider using `.find` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
personally I like `filter` because returns always an array (with elements or empty), and find could return `undefined`. So from a readability point of view, next sentences where you use the result of this call looks more evident to me.

Reply:
Your call, imo 

```
const foundElement = find()
if (foundElement) return foundElement
```

just looks as readable to me?